tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py36,py37,py38,py39,py311{py27,py36,py37,py38,py39,py311}-pandas
+envlist = py27,py36,py37,py38,py39,{py27,py36,py37,py38,py39,py311}-pandas
```

Comment:
I do not understand why we are testing until py3.9 but we do not have any CI for those. Should we consider adding those versions to the github workflow?